### PR TITLE
Completed javascript-modal exercise

### PR DESCRIPTION
**Pull request by claylevy53**
Screen Shot 2023-12-22 at 7 15 27 PM

### CODE REVIEW

**Review by kurtyazdizadeh — CHANGES_REQUESTED** [review]
While this technically works, typically a modal in practice will just show/hide existing HTML elements using javascript to toggle between CSS classes. So you could have a "show" or "hide" class on the modal that you'd toggle between using a click handler on the button to visually show the modal

If you review the README, you'll see all the needed references of HTML/CSS/JS properties to solve the challenge. 

When I assigned this exercise, we didn't do DOM creation yet.

**Comment by claylevy53** [comment]
@kurtyazdizadeh Got it, just made the necessary fixes. Let me know if this works

File: javascript-modal/main.js [hunk 1]
```diff
@@ -1,33 +1,26 @@
...
-
-  surveyButton.addEventListener('click', function (event) {
-    mainDiv.style.display = 'none';
+  $surveyButton.addEventListener('click', function (event) {
+    $modalContainer.style.display = 'none';
+    $modalStyles.style.display = 'none';
+    $overlay.style.display = 'none';
+    $row.style.display = 'none';
+    $surveyButton.style.display = 'none';
+    $modalText.style.display = 'none';
```
Comment (kurtyazdizadeh) [inline javascript-modal/main.js:24]: Though this works, there is an easier way to implement this. You could actually write a single callback function that "toggles" the applied CSS classes to the HTML element that triggered the callback.

The CSS classes you toggle will show/hide the element.

**Review by kurtyazdizadeh — APPROVED** [review]
This technically works, but there is a better way to implement the solution - I'll review it in class!

**Pull request merged**